cmd/version.go

```diff
@@ -47,10 +49,54 @@ type version struct {
 
 // versionString returns the CLI version
 func versionString() string {
-	if kubeBuilderVersion == unknown {
-		if info, ok := debug.ReadBuildInfo(); ok && info.Main.Version != "" {
+	info, ok := debug.ReadBuildInfo()
+
+	if ok && info.Main.Version != "" {
```

Comment:
Since the code becomes more complex than it was before, it would be nice to be able to write tests, mocking `debug.ReadBuildInfo()` output

Reply:
I'll definitely be interesting creating mock for it. I'll give it a shot.